playbooks/common/openshift-cluster/config.yml

```diff
@@ -11,4 +11,6 @@
 
 - include: ../openshift-master/config.yml
 
+- include: ../openshift-master/additional_config.yml
```

Comment:
Should additional_config be moved under openshift_cluster instead...  While the items are run on the first master, they are more cluster related configuration.